ros2topic/ros2topic/verb/bw.py

```diff
@@ -93,12 +97,19 @@ def __init__(self, node, window_size):
         self.sizes = []
         self.times = []
         self.window_size = window_size
+        self.use_sim_time = node.get_parameter_or(
+            'use_sim_time',
+            rclpy.Parameter('use_sim_time', rclpy.Parameter.Type.BOOL, False)).value
```

Comment:
When is `use_sim_time` not set?

Reply:
> When is `use_sim_time` not set?

self.use_sim_time will be False, no operations with `sim time`.

Reply:
False is not the same thing as not set. It should always be set.

Reply:
OK, I will update it.

Reply:
Updated.